doc/nrf/gs_recommended_versions.rst

```diff
@@ -22,6 +22,74 @@ However, there are some Zephyr features that are currently only available on Lin
 
    Before you start setting up the toolchain, install available updates for your operating system.
 
+.. _gs_supported_OS:
+
+Supported operating systems
+***************************
+
+The following table shows the operating systems and their various versions that provide support for the |NCS| tools:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Operating System
+     - x86
+     - x64
+     - ARM64
+   * - `Windows 11`_
+     - Tier 3
+     - Tier 3
+     - Not supported
+   * - `Windows 10`_
+     - Tier 1
+     - Tier 2
+     - Not supported
+   * - `Linux - Ubuntu 22.04 LTS`_
+     - Not supported
+     - Not supported
+     - Not supported
+   * - `Linux - Ubuntu 20.04 LTS`_
+     - Not supported
+     - Tier 1
+     - Not supported
+   * - `macOS 12`_
+     - Not applicable
+     - Tier 2
+     - Tier 2
+   * - `macOS 11`_
+     - Not applicable
+     - Tier 1
+     - Not supported
+   * - `macOS 10.15`_
+     - Not applicable
+     - Tier 3
+     - Not supported
+
+The table uses following tiers to categorize the level of operating system support:
+
+Tier 1
+  The |NCS| tools will always work.
+  The automated build and automated testing ensure that the application builds and successfully completes tests after each change.
+
+Tier 2
+  The |NCS| tools will always build.
+  The automated build ensures that the application builds successfully after each change.
+  There is no guarantee that a build will work because the automation tests do not always run.
+
+Tier 3
+  The |NCS| tools are supported by the application or build targets but are not built or tested by the project.
```

Comment:
Definition proposed by TPM: "The nRF Connect SDK tools are supported by design but are not built or tested by Nordic. Therefore, the application may or may not work.".

Reply:
I have rephrased the sentence. I have also rephrased the definition of Tier 1 and Tier 2 to align with the Tier 3 definition.